From eb378df238531c1d10543e2fb643cccb74a8f421 Mon Sep 17 00:00:00 2001
From: Ricardo Neri <ricardo.neri-calderon@linux.intel.com>
Date: Fri, 10 Mar 2017 12:30:20 -0800
Subject: [PATCH] chipsec: do not ship manual

Given that LUV is an automated tool we don't ship a PDF reader,
it does not make sense to include the chipsec manual. We can save
a few Kbytes of valuable ramdisk space.

Signed-off-by: Ricardo Neri <ricardo.neri-calderon@linux.intel.com>
---
 setup.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index 89bcc91..fb0da43 100644
--- a/setup.py
+++ b/setup.py
@@ -141,7 +141,7 @@ package_data = {
     "chipsec": ["VERSION", "WARNING.txt"],
     "chipsec.cfg": ["*.xml", "*.xsd"],
 }
-data_files = [("", ["chipsec-manual.pdf"])]
+data_files = []
 install_requires = []
 extra_kw = {}
 
@@ -155,7 +155,6 @@ if platform.system().lower() == "windows":
 elif platform.system().lower() == "linux":
     compression_source_files = []
     package_data["chipsec_tools.linux"] = ['*']
-    data_files = [(os.path.join("share","doc","chipsec"), ["chipsec-manual.pdf"])]
     for root, dir, path in os.walk( os.path.join( "chipsec_tools", "compression" ) ):
         for f in path:
             if os.path.splitext(f)[1][1:] == 'h':
-- 
2.9.3

